code/test/Core.Test/Naming/NamingTest.cs

```diff
@@ -207,24 +207,37 @@ public void Validate_BadFormat_StartWithNumber(string language)
         }
 
         [Fact]
-        public void DirectoryExistsValidator()
+        public void Validate_DirectoryExistsValidator_NonExistantFolderDirectory()
         {
-            var directoryValidator = new DirectoryExistsValidator(Environment.CurrentDirectory);
-            var invalidDirectoryValidator = new DirectoryExistsValidator(Path.Combine(Environment.CurrentDirectory, Guid.NewGuid().ToString()));
-            var existingDir = Directory.CreateDirectory(Guid.NewGuid().ToString());
+            var nonExistantFolderDirectoryValidator = new DirectoryExistsValidator(Path.Combine(Environment.CurrentDirectory, Guid.NewGuid().ToString()));
 
             // invalid dir
-            var invalidDirectory = invalidDirectoryValidator.Validate(Guid.NewGuid().ToString());
+            var invalidDirectory = nonExistantFolderDirectoryValidator.Validate(Guid.NewGuid().ToString());
```

Comment:
That an "invalid directory" validates as valid suggests something is wrong in this test or the name of the underlying class is very misleading.